tools/custom-roles-analyzer/README.md

```diff
@@ -67,18 +72,17 @@ This will create an executable jar inside the `target` folder which will be used
 ## Running the tool
 Run following command:
 ```
-analyze.sh --org <org_id> --format <format> --role-analysis --binding-analysis
+analyze.sh --org <org_id> --format <format> --role-analysis
 ```
 Parameters: \
 --org       : GCP Organization Id (mandatory parameter). \
 --format    : Result format (optional parameter; defaults to csv and supports json). \
 --role-analysis      : Run custom role analysis (Optional parameter; default to custom role analysis). \
---binding-analysis   : Run role binding analysis (Optional parameter; default to custom role analysis).
```

Comment:
I think this option is still in the tool. Why is it removed from here?

Reply:
IAM team advised to write blog around role-analysis tool. The blog will point to this tool. So as to avoid confusion I am commenting out documentation for binding-analysis.

I am thinking to separate binding-analysis from role analysis. When I do that I will move the binding analysis code.